just-premium-htb.js

```diff
@@ -70,6 +70,136 @@ function JustPremiumHtb(configs) {
      * Functions
      * ---------------------------------- */
 
+    function __findBid(params, bids) {
```

Comment:
Please try use request Id to match bid instead of zoneId to account for sra architecture websites.

Reply:
@ix-certification I have a question about that. From where I should get request Id? I see it on a single parcel but `__generateRequestObj` function accepts an array of them. All of passed parcels will have the same request id? Or not?

Reply:
You can pass in the returnParcel.requestId as a part of your request object and return them back in your bid response as the unique identifier. Please look at the Prelude: Configuration & Parcels section in the README for more detail.